libsolidity/AST.cpp

```diff
@@ -559,9 +560,18 @@ void VariableDeclaration::checkTypeRequirements()
 			BOOST_THROW_EXCEPTION(createTypeError("Illegal use of \"constant\" specifier."));
 		if (!m_value)
 			BOOST_THROW_EXCEPTION(createTypeError("Uninitialized \"constant\" variable."));
-		else if (m_type && !m_type->isValueType())
-			// TODO: const is implemented only for uint, bytesXX and enums types.
-			BOOST_THROW_EXCEPTION(createTypeError("Illegal use of \"constant\" specifier. \"constant\" is not implemented for this type yet."));
+		if (m_type && !m_type->isValueType())
+		{
+			// TODO: const is implemented only for uint, bytesXX, string and enums types.
```

Comment:
there is an exactly same comment in tests(SolidityEndToEndTest.cpp
), could you please add "string" there as well